site/src/github.rs

```diff
@@ -649,10 +652,36 @@ regressions or improvements in the roll up.
 
 @bors rollup=never
 @rustbot label: +S-waiting-on-review -S-waiting-on-perf",
-                    commit.sha, comparison_url
+                    commit.sha, comparison_url, summary
                 ),
             )
             .await;
         }
     }
 }
+
+async fn categorize_benchmark(commit: &database::QueuedCommit, data: &InputData) -> String {
+    let comparison = match crate::comparison::compare(
+        collector::Bound::Commit(commit.parent_sha.clone()),
+        collector::Bound::Commit(commit.sha.clone()),
+        "instructions:u".to_owned(),
+        data,
+    )
+    .await
+    {
+        Ok(Some(c)) => c,
+        _ => return String::from("ERROR categorizing benchmark run!"),
+    };
+    let summary = match crate::comparison::ComparisonSummary::summarize_comparison(&comparison) {
+        Some(s) => s,
+        None => return String::from("This benchmark run did not return any significant changes"),
```

Comment:
Can you adjust this and the summaries in general to ask users to file an issue on rustc-perf if they disagree?